pcp/PCPProcessTable.c

```diff
@@ -389,10 +389,10 @@ static bool PCPProcessTable_updateProcesses(PCPProcessTable* this) {
 
       PCPProcessTable_updateMemory(pp, pid, offset);
 
-      if ((flags & PROCESS_FLAG_LINUX_SMAPS) &&
-          (Process_isKernelThread(proc) == false)) {
-         if (Metric_enabled(PCP_PROC_SMAPS_PSS))
+      if ((flags & PROCESS_FLAG_LINUX_SMAPS) && (Process_isKernelThread(proc) == false)) {
```

Comment:
A separate commit to fix this one?

```suggestion
      if ((flags & PROCESS_FLAG_LINUX_SMAPS) && !Process_isKernelThread(proc)) {
```

Reply:
Can do …